operators/multiclusterobservability/controllers/multiclusterobservability/observatorium.go

```diff
@@ -377,6 +377,7 @@ func newRuleSpec(mco *mcov1beta2.MultiClusterObservability, scSelected string) o
 	} else {
 		ruleSpec.Retention = mcoconfig.RetentionInLocal
 	}
+	ruleSpec.EvalInterval = fmt.Sprintf("%ds", mco.Spec.ObservabilityAddonSpec.Interval)
```

Comment:
I think we need to expose it in MCO CRD. so here if there is no defined in MCO CR, then set mco.Spec.ObservabilityAddonSpec.Interval as default value. if so, use the customized value.

Reply:
done.